hazelcast/src/main/java/com/hazelcast/instance/impl/HazelcastBootstrap.java

```diff
@@ -259,8 +259,9 @@ public static synchronized HazelcastInstance getInstance() {
     private static HazelcastInstance createStandaloneInstance() {
         configureLogging();
         LOGGER.info("Bootstrapped instance requested but application wasn't called from hazelcast submit script. "
-                + "Creating a standalone Hazelcast instance instead.");
+                + "Creating a standalone Hazelcast instance instead. Jet is enabled in this standalone instance.");
         Config config = Config.load();
+        config.getJetConfig().setEnabled(true);
```

Comment:
The bootstrap instance is used when developing a job to be submitted to a cluster - most commonly started from the zip distribution.

So I thinkg the config should be same as the zip distribution from security point of view - namely Jet enabled & bind network to local host only - otherwise starting from IDE opens the developers machine to anyone on the same network.

Reply:
before the default was multicast and it was not meant to form a cluster with other nodes.

Reply:
That's still there: https://github.com/hazelcast/hazelcast/pull/19080/files/db8705d5bf43d7233889c41bbfea4d846bb14f51#diff-9c6590016fd6dae964e3b0d9d749e9327f3bf56c102f4f47cdb86ceee83aa1e4R270-R279

My point is we should forbid non-local clients connecting to the bootstrap instance.

Reply:
The loopback-only bind is used in the new commits.